Cargo.toml

```
@@ -35,9 +35,9 @@ features = ["v4"]
 version = "0.4"
 features = ["kv_unstable", "serde", "std", "kv_unstable_serde", "kv_unstable_std"]
 
-[dependencies.chrono]
-version = "0.4"
-features = ["serde"]
+[dependencies.time]
+version = "0.3"
+features = ["parsing", "serde", "serde-human-readable"]
```

one change I would make here

```suggestion
features = ["parsing", "serde", "formatting"]
```

This works and doesn't make use of a deprecated feature.

forget everything else I said, this will do just fine.